src/main/webapp/data-visualization.js

```diff
@@ -306,38 +301,24 @@ function setStyle(isCountyQuery) {
   mapsDiv.style.display = 'none';
 }
 
-function initDataTable(isCountyQuery) {
-  const table = document.createElement('table');
-  table.innerHTML = '';
-  const tableRow = document.createElement('tr');
-  const region = document.createElement('th');
-  const value = document.createElement('th');
-  region.innerText = isCountyQuery ? 'County' : 'State';
-  value.innerText = 'Population';
-  tableRow.appendChild(region);
-  tableRow.appendChild(value);
-  table.appendChild(tableRow);
-  return table;
-}
-
-/**
- * Create and return an HTML table using the data in dataArray.
- * Uses first row in dataArray as table headers.
- */
-function appendRowToDataTable(table, locationName, population) {
-  const tableRow = document.createElement('tr');
-  const region = document.createElement('td');
-  const value = document.createElement('td');
-  region.innerText = locationName;
-  value.innerText = population;
-  tableRow.appendChild(region);
-  tableRow.appendChild(value);
-  table.appendChild(tableRow);
+// Draw data table using Visualization API
+function drawTable(dataArray, isCountyQuery) {
+  google.charts.load('current', {'packages': ['table']});
+  google.charts.setOnLoadCallback(() => {
+    const data = new google.visualization.DataTable();
+    const nameHeader = isCountyQuery ? 'County' : 'State';
+    data.addColumn('string', nameHeader);
+    data.addColumn('number', 'Population');
```

Comment:
We probably want to change this text to be something more dynamic, since it's not always Population, but that can be a later job

Reply:
It would be pretty easy to pass in description in this function from displayVisualization